src/ResourceManager/Network/Commands.Network/help/Get-AzureRMNetworkWatcherReachabilityReport.md

```diff
@@ -0,0 +1,297 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRMNetworkWatcherReachabilityReport
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
```

Comment:
@irrogozh please fill in any templates in the markdown files you've added

Reply:
Done